test/controllers/security.test.js

```diff
@@ -1003,9 +1003,7 @@ describe('Security Controller', () => {
               controller: 'security',
               action: 'searchProfiles',
               body: {roles: ['foo', 'bar']},
-              from: undefined,
-              size: undefined,
-              scroll: undefined
+              opt: 'in'
```

Comment:
I personally prefer to add a test than to modify an existing one even if you copy/paste it